_includes/tutorials/aggregating-count/ksql/code/docker-compose.yml

```diff
@@ -35,15 +35,16 @@ services:
     hostname: schema-registry
     container_name: schema-registry
     depends_on:
+      - zookeeper
       - broker
     ports:
       - "8081:8081"
     environment:
       SCHEMA_REGISTRY_HOST_NAME: schema-registry
-      SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: 'broker:9092'
+      SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL: 'zookeeper:2181'
```

Comment:
Why are we introducing ZK as a dependency here? These configs were swapped a while back in [this PR](https://github.com/confluentinc/kafka-tutorials/pull/590) to address [this issue](https://github.com/confluentinc/kafka-tutorials/issues/562).

Reply:
Thanks for pointing this out. It looks like a previous merge had unintentionally wiped these changes out. I cherry-picked the pr merge commit back into this branch.

Reply:
Thanks! The latest commit looks great!